### PR TITLE
update foxglove header/footer menu links on mcap website

### PR DESCRIPTION
**Pull request by admfrk**
### Public-Facing Changes
add a foxglove link to the mcap website header
update foxglove links in the mcap website footer

### Description
post foxglove 2.0 foxglove.dev URLs have changed

n/a

### CODE REVIEW

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -111,6 +111,11 @@ const config = {
             position: "left",
             label: "Specification",
           },
+          {
+            href: "https://foxglove.dev",
+            label: "Foxglove",
+            position: "right",
```
Comment (jtbandes) [inline website/docusaurus.config.js:117]: This CTA doesn't seem very clear about why one would want to click on Foxglove from the mcap site?

Comment (amacneil) [reply website/docusaurus.config.js:117]: +1

File: website/docusaurus.config.js [hunk 2]
```diff
@@ -111,6 +111,11 @@ const config = {
...
+          },
```
Comment (amacneil) [inline website/docusaurus.config.js:118]:
```suggestion
```

File: website/docusaurus.config.js [hunk 3]
```diff
@@ -168,8 +168,8 @@ const config = {
             title: "Enterprise",
             items: [
               {
-                label: "Foxglove Data Platform",
-                href: "https://foxglove.dev/data-platform",
+                label: "Foxglove",
+                href: "https://foxglove.dev/product",
```
Comment (amacneil) [inline website/docusaurus.config.js:172]:
```suggestion
                href: "https://foxglove.dev/",
```

**Review by amacneil — APPROVED** [review]

**Pull request merged**